nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfigEdit.js

```diff
@@ -774,13 +775,37 @@ export function RanEdit(props: Props) {
   const [connectedEnodebs, setConnectedEnodebs] = useState<enodeb_serials>(
     props.gateway.connected_enodeb_serials,
   );
+  const [unregisteredEnodebs, setUnregisteredEnodebs] = useState({});
```

Comment:
minor : This can be a newUnregisteredEnbInfo and can be a list

Reply:
Done.